### PR TITLE
Feat/redesign start screen no account

### PR DESCRIPTION
**Pull request by Nokiusz**
This PR consists of redesigning `1.0.0 Start screen no account`

### CODE REVIEW

File: packages/extension-ui/src/components/HelperFooter.tsx [hunk 1]
```diff
@@ -9,29 +9,30 @@ import styled from 'styled-components';
...
 }
 
-function HelperFooter({ children, className }: Props): React.ReactElement<Props> {
+function HelperFooter({ children, className, tooltipOnly }: Props): React.ReactElement<Props> {
   return <div className={className}>{children}</div>;
 }
 
 export default styled(HelperFooter)(
-  ({ theme }: Props) => `
+  ({ theme, tooltipOnly }: Props) => `
```
Comment (jalooc) [inline packages/extension-ui/src/components/HelperFooter.tsx:20]: Could you explain how this relates to any kind of a tooltip? I was unable to infer that from the code, but maybe I missed something 🧐

Comment (Nokiusz) [reply packages/extension-ui/src/components/HelperFooter.tsx:20]: image
it is referred as a tooltip on the designs so I went with that.
I'm open to suggestions tho 👯

Comment (jalooc) [reply packages/extension-ui/src/components/HelperFooter.tsx:20]: Oh, I think it just clicked for me - _tooltip only_ was meant to indicate that there's nothing under that component and no margin is necessary to push the content down?

I'd then suggest a different, more scalable approach, however requiring more work/code so up to you whether to employ it :) Namely, any properties influencing _external layout / position_ should be imposed by the consumers of the component in question - after all, the component itself doesn't know where or how it's laid out in the application. E.g. ``const Consumer = styled(TheComponent)`margin-bottom: 16px;'` `` where needed.

Comment (jalooc) [reply packages/extension-ui/src/components/HelperFooter.tsx:20]: Just explaining a why I believe that ☝️ is a good practice: This enables you to lay out the component in virtually any environment, without need o changing its internals and thus potentially adapting all other usages and/or growing it's interface with usage-specific cases. You want it spaced with a margin - you got it; you want it spaced with a gap in a grid - you got it; you want it not spaced at all - still got it; none of those cases requiring changes in the component, keeping it truly reusable.

Comment (Nokiusz) [reply packages/extension-ui/src/components/HelperFooter.tsx:20]: Sounds great! I'll try to adapt to that 😄

**Pull request merged**